charts/fluentd/values.yaml

```diff
@@ -3,3 +3,8 @@ pull_policy: "Always"
 docker_tag: canary
 # limits_cpu: "100m"
 # limits_memory: "50Mi"
+
+# external syslog endpoint
+# syslog:
+#  syslog_host: "external_syslog_endpoint_url"
```

Comment:
uncomment these and keep the value as empty with the help text as comments as the helm fails the above condition if it can't find the keys.